apps/api/test/comments.e2e-spec.ts

```diff
@@ -0,0 +1,361 @@
+        .variables(QUERY_ANNOTATION_COMMENTS_VARIABLES)
+        .expectNoErrors();
+
+      expect(dataQuery).toMatchSnapshot();
+
+      expect(data).toMatchSnapshot();
+    });
+
+    it('Annotation owner can create a new comment', async () => {
+      const { data } = await request(app.getHttpServer())
+        .auth(`${UsersData.sessionParticipant1.id}`, { type: 'bearer' })
+        .mutate(CREATE_COMMENT)
+        .variables(CREATE_COMMENT_VARIABLES_ANNOTATION_OWNER)
+        .expectNoErrors();
+
+      expect(data).toMatchSnapshot();
+
+      const { data: dataQuery } = await request(app.getHttpServer())
+        .auth(`${UsersData.sessionParticipant1.id}`, { type: 'bearer' })
+        .query(QUERY_ANNOTATION_COMMENTS)
+        .variables(QUERY_ANNOTATION_COMMENTS_VARIABLES)
+        .expectNoErrors();
+
+      expect(dataQuery).toMatchSnapshot();
+    });
+
+    it('Authenticated user who is participant in the session but not the annotation owner should not be allowed to create a comment', async () => {
+      const { errors } = await request(app.getHttpServer())
+        .auth(`${UsersData.sessionParticipant2.id}`, { type: 'bearer' })
+        .mutate(CREATE_COMMENT)
+        .variables(CREATE_COMMENT_VARIABLES_ANNOTATION_OWNER);
+
+      expect(errors).toHaveLength(1);
+      expect(errors[0].message).toBe('Forbidden');
+    });
+  });
+
+  describe('Update Comment', () => {
+    it('Not authenticated user should get "Unauthorized" error', async () => {
+      const { errors } = await request(app.getHttpServer())
+        .mutate(UPDADTE_COMMENT)
+        .variables(UPDATE_COMMENT_VARIABLES_SESSION_OWNER);
+
+      expect(errors).toHaveLength(1);
+      expect(errors[0].message).toBe('Unauthorized');
+    });
+
+    it('Session Owner can update its own comment', async () => {
+      const { data } = await request(app.getHttpServer())
+        .auth(`${UsersData.sessionOwner1.id}`, { type: 'bearer' })
+        .mutate(UPDADTE_COMMENT)
+        .variables(UPDATE_COMMENT_VARIABLES_SESSION_OWNER)
+        .expectNoErrors();
+
+      expect(data).toMatchSnapshot();
+    });
+
+    it('Annoation Owner can update its own comment', async () => {
+      const { data } = await request(app.getHttpServer())
+        .auth(`${UsersData.sessionParticipant1.id}`, { type: 'bearer' })
+        .mutate(UPDADTE_COMMENT)
+        .variables(UPDATE_COMMENT_VARIABLES_ANNOTATION_OWNER)
+        .expectNoErrors();
+
+      expect(data).toMatchSnapshot();
+    });
+
+    it('Annoation Owner cannot update session owners comment', async () => {
+      const { errors } = await request(app.getHttpServer())
+        .auth(`${UsersData.sessionParticipant1.id}`, { type: 'bearer' })
+        .mutate(UPDADTE_COMMENT)
+        .variables(UPDATE_COMMENT_VARIABLES_SESSION_OWNER);
+
+      expect(errors).toHaveLength(1);
+      expect(errors[0].message).toBe('Forbidden');
+    });
+
+    it('Session Owner cannot update session owners comment', async () => {
```

Comment:
Description copy paste error, session owner cannot update participant's comments but of course the session owner comments